### ISSUE TITLE
Can you add server information and online client IP port information to mop browser?

### ISSUE THREAD

**Issue by allnode-github** [streamnative/mop#146]
Can you add server information and online client IP port information to mop browser?

**Comment by codelipenghui** [comment]
@allnode-github Yes, we are planning to add REST API extension for MoP such as metrics endpoint, management endpoint etc..

**Comment by Technoboy-** [comment]
Hi, after #170 merged, you can use rest API to visit mop stats.
Rest path is : pulsar-webaddress/mop-stats
